--- linux.orig/drivers/pcmcia/Config.in
+++ linux/drivers/pcmcia/Config.in
@@ -22,6 +22,7 @@
       dep_tristate '  HD64465 host bridge support' CONFIG_HD64465_PCMCIA $CONFIG_PCMCIA
    fi
    dep_bool '  i82092 compatible bridge support' CONFIG_I82092 $CONFIG_PCI
+   dep_bool '  PLX9052 host bridge support' CONFIG_PLX9052 $CONFIG_PCI
    bool '  i82365 compatible bridge support' CONFIG_I82365
    if [ "$CONFIG_ARCH_SA1100" = "y" ]; then
       dep_tristate '  SA1100 support' CONFIG_PCMCIA_SA1100 $CONFIG_PCMCIA
--- linux.orig/drivers/pcmcia/Makefile
+++ linux/drivers/pcmcia/Makefile
@@ -32,6 +32,9 @@
   ifeq ($(CONFIG_I82092),y)
     obj-y += i82092.o
   endif
+  ifeq ($(CONFIG_PLX9052),y)
+    obj-y += plx9052.o
+  endif
   ifeq ($(CONFIG_TCIC),y)
     obj-y += tcic.o
   endif
@@ -47,6 +50,9 @@
     ifeq ($(CONFIG_I82092),y)
       obj-m += i82092.o
     endif
+    ifeq ($(CONFIG_PLX9052),y)
+      obj-m += plx9052.o
+    endif
     ifeq ($(CONFIG_TCIC),y)
       obj-m += tcic.o
     endif
